src/terrain_management/terrain_manager.py

```diff
@@ -215,7 +217,6 @@ def renderMesh(
             update_topology (bool): whether to update the mesh topology."""
 
         mesh = UsdGeom.Mesh.Get(self._stage, self._mesh_path)
-        self._mesh_path = self._og_mesh_path + "_" + str(self._id)
```

Comment:
You need to be careful here, I don't know if they fixed it. But in 2022.2.1 their was an issue with computing colliders of mesh whose geometry changed. In short, the collider is cached and if the name is the same, then they do not recompute it. Please check that colliders are now updated properly before changing this line. I did raise an issue to them and they said it would be fixed soon, but I don't know if it ever has been.

Reply:
Yes you are right and fixed this part. 
2023.1.1 still has the issue about caching collider.